drivers/dai/intel/ssp/dai-params-intel-ipc4.h

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __DAI_PARAMS_INTEL_IPC4_H__
+#define __DAI_PARAMS_INTEL_IPC4_H__
+
+#include <stdint.h>
+
+#define I2S_TDM_INVALID_SLOT_MAP1 0xF
```

Comment:
This could collide with other I2S TDM headers/users. Can you change prefix to ```DAI_INTEL_TDM_...```

Reply:
not needed, removed.